File: 0412_FizzBuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> list[str]:
        ans = []
        for i in range(1, n+1):
            st = ""
            if not i%3:
                st += "Fizz"
            if not i%5:
                st += "Buzz"
            if st:
                ans.append(st)
            else:
                ans.append(str(i))
        return ans

File: 0412_FizzBuzz.go
func fizzBuzz(n int) []string {
    var mySlice []string
    for i := 1; i <= n;i++ {
        s := ""
        if i%3 == 0 {
            s += "Fizz"
        }
        if i%5 == 0 {
            s += "Buzz"
        }
        if s != "" {
            mySlice = append(mySlice, s)
        } else
        {
            mySlice = append(mySlice, strconv.Itoa(i))
        }
    }
    return mySlice
}